modules/ROOT/pages/fd-reuse-config.adoc

```diff
@@ -6,11 +6,13 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can reuse a connection in a Flow Designer project instead of adding your authentication information manually. After a reusable connection is created, it is available for all Flow Designer projects in your business group. Changing your resuable configuration is can be done anytime. 
+You can reuse a connection in a Flow Designer project instead of manually adding your authentication information.
+After a reusable connection is created, it is available for all Flow Designer projects in your business group.
+You can change your reusable configuration at anytime.
 
 You can reuse connections:
```

Comment:
```suggestion
You can reuse connections in the following circumstances:
```
I think we need this addition because the list items do not naturally flow, grammatically, from the shorter intro.

Reply:
Fixed.